design-proposals/dedicated-migration-network.md

```diff
@@ -0,0 +1,94 @@
+# Overview
+This design provides a way to migrate VMIs (Virtual Machine Instances) over a dedicated network.
+VMI migrations currently happen over the main Kubernetes network all pods are connected to.
+
+## Motivation
+In order for VMI migrations to succeed, there must be a fair amount of bandwidth available.
+If for example the memory of a migrating VMI changes faster than the current speed of the main network, the migration will never succeed.
+This is especially problematic during mass eviction events like platform upgrades.
+This problem is currently mitigated by throttling the number of concurrent migrations and using long timeouts.
+
+## Goals
+- Allow a cluster admin to setup and dedicate a secondary physical cluster network to migrations.
+- Keep the main network as the default medium for migrations.
+
+## Non Goals
+- Network configuration. A functional secondary physical network should be provided.
+- Network definition. A valid NetworkAttachmentDefinition should be written by the cluster admin and provided to the KubeVirt CR (Custom Resource).
+- Cluster components. A multi-network CNI like multus should be properly installed, as well as necessary plugins, and any required daemon (such as CNI DHCP) should be running on each node.
```

Comment:
The mentioning of `CNI DHCP` can now be dropped.
I think you can just mention "daemonsets" in general, avoiding details.